megamek/i18n/megamek/common/messages.properties

```diff
@@ -451,4 +451,6 @@ MovementType.StationKeeping=Station Keeping
 MiscType.drones=drones
 MiscType.drone=drone
 MiscType.theaters=theaters
-MiscType.theater=theater
\ No newline at end of file
+MiscType.theater=theater
+MiscType.tons=tons
```

Comment:
Did we really not have a token for "ton" and "tons?

Reply:
We might. I didn'tcheck because I thought the advantages of keeping this in one place were worth the possibility of duplication.